xray/src/generation.rs

```diff
@@ -545,6 +543,7 @@ pub fn build_xray_quadtree(
     let (all_nodes_tx, all_nodes_rx) = mpsc::channel();
     println!("Building level {}.", deepest_level);
 
+    let global_from_local = &local_from_global.clone().map(|t| t.inverse());
```

Comment:
Naive question: Why is this clone() needed?

Reply:
We create a reference to a temporary here, without clone this wouldn't be possible.

Reply:
What do you think about using `as_ref()`? Not that performance matters here, but it feels slightly cleaner.

Reply:
Yep, good idea, done.